libraries/AP_Logger/AP_Logger.cpp

```diff
@@ -56,6 +56,14 @@ extern const AP_HAL::HAL& hal;
 #define HAL_LOGGER_ARM_PERSIST 15
 #endif
 
+#ifndef HAL_LOGGER_MIN_MB_FREE
+#if AP_FILESYSTEM_LITTLEFS_ENABLED
+#define HAL_LOGGER_MIN_MB_FREE 1
```

Comment:
Probably needs to be bigger so that a meaningful amount can be logged. Also to match the parameter range annotation.

Reply:
Actually, this needs to change so that the oldest file is deleted if there is not enough space, not sure that works right now

Reply:
Ok the code exists, I have massaged this to be more appropriate